internal/utils/codecrafters_client.go

```diff
@@ -190,6 +199,9 @@ func (c CodecraftersClient) FetchBuild(buildId string) (FetchBuildStatusResponse
 			}
 
 			if fetchBuildResponse.Status != "failure" && fetchBuildResponse.Status != "success" {
+				customError := &CustomError{Message: "unexpected submission status: " + fetchBuildResponse.Status}
+				sentry.CaptureException(customError)
```

Comment:
Why is `CustomError` important here? If it's just carrying `Message`, can't we use `fmt.Errorf`?